harvard-lts/drs-migration-dashboard
# Line graph + style changes

**Line graph + style changes**
* * *

**GitHub Issue**: (https://github.com/harvard-lts/drs-migration-dashboard/issues/7) 

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)

# What does this Pull Request do?
Introduces line graphs as a way to track progress and incorporated design feedback based on meeting with Janet

# How should this be tested?
Test page: https://localhost:3001/migrationstatus/piechart/

Line graphs
* Click on the "trends" tab in each card to see line graphs (used js to create arrays of the dates + totals remaining, then charts.js to create the line)
* Also did a little math to change the y-axis to TB/MB/KB accordingly, otherwise the y-axis was showing alllll the zeros and taking up far too much space in the card
* When cards are 3-across, the graphs are quite small and illegible, so added a "click to enlarge" link that opens up the line graph in a larger modal window (a suggestion from Janet)

Design updates
* Removed large title to push the dashboard content up higher on the page (header now says "DRS Migration Dashboard" and acts as the h1 for the page)
* Revised language and restyled the "last update" line to make more prominent
* Changed page's max container width so that cards could be larger (also set a min-width for the cards so that the graphs and doughnut charts can't get too small; cards now wrap nicely on the page)
* Updated colors for doughnut graphs (success is now a bright green, verified is teal, all others are the same as before)
* Updated tables in the numbers view (slightly smaller font-size, used Trueno light font for readability, numbers right-aligned)
* Updated definitions modal window to use Trueno font and added more padding between definitions

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? no
- integration tests? no

# Interested parties
@dl-maura @phil-plencner-hl @awoods
